### PR TITLE
Parameters from orm.xml moved to the TreeBuilder

### PR DESCRIPTION
**Pull request by quentinsikana**

### CODE REVIEW

File: Resources/config/orm.xml [hunk 1]
```diff
@@ -4,12 +4,14 @@
...
         http://symfony.com/schema/dic/services/services-1.0.xsd">
 
+<!--
     <parameters>
         <parameter key="asm_translation_loader.translation_manager.class">Asm\TranslationLoaderBundle\Doctrine\TranslationManager</parameter>
         <parameter key="asm_translation_loader.translation_history_manager.class">Asm\TranslationLoaderBundle\Doctrine\TranslationHistoryManager</parameter>
         <parameter key="asm_translation_loader.model.translation.class">Asm\TranslationLoaderBundle\Entity\Translation</parameter>
         <parameter key="asm_translation_loader.model.translation_history.class">Asm\TranslationLoaderBundle\Entity\TranslationHistory</parameter>
     </parameters>
+ -->
```
Comment (xabbuh) [inline Resources/config/orm.xml]: Can you just remove these lines please?

**Comment by xabbuh** [comment]
@quentinsikana Thanks for this PR. The changes look valid to (we should probably get rid of the class parameters in a future release, but that's nothing we can change right now). Can you also update the tests so that they pass again? Adding another test case that makes sure that we can actually change the parameters through the config would be great.

**Comment by quentinsikana** [comment]
Sure!
But I must confess I'm still not familiar with Symfony2 testing. Could you give me an idea on how to do this?

**Comment by quentinsikana** [comment]
@xabbuh Sure!
But I must confess I'm still not familiar with Symfony2 testing. Could you give me an idea on how to do this?

**Comment by quentinsikana** [comment]
@xabbuh Old tests are now passing and I added a new one as you suggested.

**Comment by quentinsikana** [comment]
Everything looks fine, let's merge? :)

**Comment by xabbuh** [comment]
Thank you @quentinsikana. I have squashed your changes during merging. That's why your pull request is shown as closed instead of merged. I also added an entry to the changelog in ec515fb2a3c0b273f2e3e6e0e15ce2b2a11f2c3d. Thanks for your contribution!

**Pull request closed without merging**